catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/transformer/mgmp/MgmpTransformer.java

```diff
@@ -0,0 +1,604 @@
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_DATUM_WKT_TYPE_PATH,
+
+                MgmpConstants.ISR_COVERAGE_CATEGORY_PATH,
+                MgmpConstants.ISR_COVERAGE_COMMENT_PATH,
+                MgmpConstants.ISR_IMAGE_COMMENT_PATH,
+                MgmpConstants.ISR_IMAGE_DESCRIPTION_PATH,
+                MgmpConstants.ISR_MD_IMAGE_COMMENT_PATH,
+                MgmpConstants.ISR_MD_IMAGE_DESCRIPTION_PATH,
+                MgmpConstants.ISR_VIDEO_COMMENT_PATH,
+                MgmpConstants.ISR_VIDEO_DESCRIPTION_PATH,
+
+                MgmpConstants.METADATA_ORIGINATOR_SECURITY_PATH,
+                MgmpConstants.METADATA_RELEASABILITY_PATH,
+                MgmpConstants.RESOURCE_SECURITY_RELEASABILITY_PATH,
+                MgmpConstants.NIIRS_PATH,
+                MgmpConstants.NIIRS_RATING_PATH,
+                MgmpConstants.RESOURCE_SECURITY_PATH,
+                MgmpConstants.METADATA_SECURITY_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_CRS_WKT_TYPE_PATH)
+                .stream()
+                .map(this::toPath)
+                .forEach(paths::add);
+
+        paths.addAll(super.buildPaths());
+        return paths;
+    }
+
+    @Override
+    public MetacardImpl toMetacard(final XstreamPathValueTracker pathValueTracker, String id) {
+        MetacardImpl metacard = new MetacardImpl(mgmpMetacardType);
+        super.setMetacardAttributes(metacard, id, pathValueTracker);
+        addMetacardIsrInformation(pathValueTracker, metacard);
+        addMetacardNiirsInformation(pathValueTracker, metacard);
+        addMetacardLanguage(pathValueTracker, metacard);
+        addMetacardCloudCoverPercentage(pathValueTracker, metacard);
+        addMetacardResourceSecurityClassification(pathValueTracker, metacard);
+        addMetacardResourceSecurityReleasability(pathValueTracker, metacard);
+        addMetacardResourceSecurityOriginatorClassification(pathValueTracker, metacard);
+        addMetacardMetadataSecurityReleasability(pathValueTracker, metacard);
+        addMetacardMetadataSecurityOriginatorClassification(pathValueTracker, metacard);
+        addMetacardMetadataSecurityClassification(pathValueTracker, metacard);
+        addMetacardDataQuality(metacard);
+        metacard.setAttribute(Media.TYPE, MgmpConstants.DEFAULT_MEDIA_TYPE);
+        return metacard;
+    }
+
+    @Override
+    public void addMetacardFormat(final XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        String format = pathValueTracker.getFirstValue(toPath(MgmpConstants.FORMAT_PATH));
+        String formatVersion =
+                pathValueTracker.getFirstValue(toPath(GmdConstants.FORMAT_VERSION_PATH));
+        if (StringUtils.isNotEmpty(format)) {
+            metacard.setAttribute(Media.FORMAT, format);
+        }
+
+        if (StringUtils.isNotEmpty(formatVersion)) {
+            metacard.setAttribute(Media.FORMAT_VERSION, formatVersion);
```

Comment:
It probably doesn't make sense to set the format version if you are not setting the format